content/docs/cloud/concepts.md

```diff
@@ -3,51 +3,51 @@ title: Concepts
 
 Compute is stateless and can be automatically activated and suspended due to user activity.
 
-### Compute Lifecycle
+## Compute Lifecycle
 
 A Compute node in Neon is a stateless Postgres process due to the separation of storage and compute. It has two main states: Active and Idle.
 
-Active means that Postgres is running right now. If there are no active queries for 5 minutes, the activity monitor will gracefully put the corresponding compute node into the idle state to save energy and resources. The activity monitor is conservative, and it treats 'idle in transaction' connections as some activity to do not break an application logic that relies on long-lasting transactions. Yet, it closes all 'idle' connections after 5 minutes of complete inactivity.
+Active means that Postgres is running currently. If there are no active queries for 5 minutes, the activity monitor will gracefully put the corresponding compute node into the Idle state to save energy and resources. The activity monitor is conservative, and it treats 'idle in transaction' connections as some activities do not break an application logic that relies on long-lasting transactions. Yet, it closes all 'idle' connections after 5 minutes of complete inactivity.
 
-In the Idle state, you can still connect to your compute at any time. Neon will automatically activate it. Activation usually happens within a few seconds, so the first connection in the idle state will have higher latency. Also, the Postgres page cache (shared buffers) will be cold after waking up from idle state, and your usual queries may take longer.
+In the Idle state, you can still connect to your compute at any time. Neon will automatically activate it. Activation usually happens within a few seconds, so the first connection in the Idle state will have higher latency. Also, the Postgres page cache (shared buffers) will be cold after waking up from Idle state, and your usual queries may take longer.
 
-After some period in the idle state, Neon will start occasionally activating your Compute to check data availability. The checks period will gradually increase up to several days if the Compute does not receive any client connections.
+After some period in the Idle state, Neon will start occasionally activating your Compute to check data availability. The checks period will gradually increase up to several days if the Compute does not receive any client connections.
 
-You can check all Compute state transitions in the operations list tab on the dashboard.
+You can check all Compute state transitions in the Operations List tab on the dashboard.
 
-### Postgres Users
+## Postgres Users
 
-Postgres users are created as part of Neon Project.
+Postgres users are created as a part of your Neon Project.
```

Comment:
FYI, I've added some new content for the Postgres Users secion in a PR and added you as a reviewer, @alexandriastech